subworkflows/nf-core/fastq_align_bwa/main.nf

```diff
@@ -0,0 +1,41 @@
+//
+// Alignment with BWA
+//
+
+include { BWA_MEM                 } from '../../../modules/nf-core/bwa/mem/main'
+include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
+
+workflow FASTQ_ALIGN_BWA {
+    take:
+    ch_reads // channel (mandatory): [ val(meta), [ reads ] ]
+    ch_index // channel (mandatory): [ val(meta2, [ index ] ]
```

Comment:
Add `ch_fasta` here too to pass to `BAM_SORT_STATS_SAMTOOLS` if required.

Reply:
Addressed [ff530ff](https://github.com/nf-core/modules/pull/2451/commits/ff530ff3cf56818d22f32d93b1123bbbc4462f28)